core/crypto/src/bls.rs

```diff
@@ -215,15 +215,11 @@ impl BlsSignature {
     }
 
     pub fn verify_aggregate(&self, data: &[u8], public_keys: &[BlsPublicKey]) -> bool {
-        if !cfg!(feature = "fake_crypto") {
```

Comment:
let's remove then feature from Cargo.toml's as well and above from verify_single.
And from using it in gitlab.yaml